### ISSUE TITLE
After the new upgrade "color" function isn't working. Same code was working prior to the update. 

### ISSUE THREAD

**Issue by dmaccccc** [davidgohel/flextable#99]
After the new upgrade "color" function isn't working. Same code was working prior to the update.

**Comment by davidgohel** [comment]
The new version is causing an issue for users of rstudio viewer on Windows. This is ok when using a standard navigator (chrome, safari, firefox). Are you in this case?

**Comment by davidgohel** [comment]
I just pushed a fix (on github). It should be ok now. Sorry for that!

It will be submitted to cran end of next week. Meanwhile, you can use github version. 

David

**Comment by Nova-Scotia** [comment]
Tested this morning, working just fine. I was having the same issue as @dmaccccc and I can now see colors in the RStudio environment in my table. This is a very timely fix for me. Thanks so much @davidgohel !

**Comment by davidgohel** [comment]
@Nova-Scotia thanks for your feedback, this helps

**Comment by davidgohel** [comment]
Hi @martino165 

This is because you did not assign the result in x in your anonymous function. 

```
index <- which(dat[ , get(colnm)>= 3])
x <- color(...
```

This was an unwanted side effect and has been solved in the latest version.

```
library(flextable)
library(data.table)
library(magrittr)

dat <- as.data.table(cbind(A=1:5,B=3:7))

rt <- regulartable(
  data = dat) %>%
  theme_booktabs() %>%
  autofit() %>%
  (
    function(x){
      for (colnm in names(dat)){
        index <- which(dat[ , get(colnm)>= 3])
        x <- color(x, i = index, j = colnm, color = "red")
      }
      return(x)
    }
  )
```

**Comment by davidgohel** [comment]
The fix is on cran now in the new version `0.5.1`.
